### PR TITLE
New Presence: Ekrem İmamoğlu & Sürat Kargo

### PR DESCRIPTION
**Pull request by erdem85**
### Sürat Kargo
LgByEauWI2

### Ekrem İmamoğlu
9xpAZmwSeS

> [Ekrem İmamoğlu](https://www.ekremimamoglu.com/)
> [Sürat Kargo](https://www.suratkargo.com.tr/)

### CODE REVIEW

**Comment by MusaBrt** [comment]
you're shining dude

File: websites/S/Sürat Kargo/presence.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    presenceData.details = "İnsan Kaynakları Politikamız";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path.startsWith("/SosyalSorumlulukCalismalarimiz")){
+    presenceData.details = "Sosyal Sorumluluk Çalışmalarımız";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path == "/"){
+    presenceData.details = "Ana Sayfa";
+    delete presenceData.state;
```
Comment (Bas950) [inline websites/S/Sürat Kargo/presence.ts:89]: why are you doing delete state when its not even set?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/Sürat Kargo/presence.ts (hunk 1 above)
Comment (erdem85) [reply websites/S/Sürat Kargo/presence.ts]: fixed

**Comment by erdem85** [comment]
i added 1 more presence.

**Review by ririxidev — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Comment by erdem85** [comment]
> dont upload js file

i deleted js file before

 ( https://github.com/PreMiD/Presences/pull/2484/commits/52703ff116441582c7183a92a9b9de5d00f4c94a )

**Review by Bas950 — APPROVED** [review]

**Pull request merged**